src/base/bittorrent/sessionimpl.cpp

```diff
@@ -5530,12 +5530,14 @@ void SessionImpl::handlePeerBlockedAlert(const lt::peer_blocked_alert *p)
         break;
     case lt::peer_blocked_alert::port_filter:
         reason = tr("port filter", "this peer was blocked. Reason: port filter.");
+        reason.append(u" (%1)"_qs.arg(QString::number(p->endpoint.port())));
```

Comment:
Why not modify existing reason string? E.g.:
```c++
reason = tr("port %1 is filtered", "this peer was blocked. Reason: port 8899 is filtered.").arg(QString::number(p->endpoint.port()));
```

Reply:
It will mess with localization, requiring to update the string in all languages. I tried to avoid it.

Reply:
AFAIK, Right-to-left languages will expect the number is shown in the left most position: `(12345) port filter`.
It seems accepting @glassez suggestion is the easy way to avoid it and stay in line with other messages.

Reply:
How should I do that?
Edit every `.ts` file, replacing `<source>` part of a string, leaving `<translation>` to be updated later?

Reply:
You shouldn't touch translation files. Just do what I suggested above. Then translators will be able to adjust format according to the rules of their language.

Reply:
Ok.